examples/vmcp-config.yaml

```diff
@@ -0,0 +1,104 @@
+# Virtual MCP Server (vmcp) Configuration Example
+# This file demonstrates the configuration format for vmcp
```

Comment:
this configuration is not conformant with the proposal, let's not add it just yet.